packages/html-entities/src/index.ts

```diff
@@ -57,5 +57,5 @@ export function decodeEntities( html ) {
 	 *
 	 * @see https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent
 	 */
-	return /** @type {string} */ ( decoded );
+	return /** @type {string} */ decoded;
```

Comment:
This type is no longer needed

```suggestion
	return decoded;
```